src/components/pages/donation_confirmation/SuccessMessageBankTransfer.vue

```diff
@@ -2,18 +2,19 @@
 	<div class="donation-confirmation-card success-message-bank-transfer">
 		<div class="columns is-multiline pt-0 pb-0 mt-0 mb-0">
 			<div class="column is-full pt-0 pb-0 mt-0 mb-0">
-				<h1 class="icon-title"><warning-icon/> {{ $t( 'donation_confirmation_topbox_payment_title_bank_transfer_alt' ) }}</h1>
+				<h1 class="icon-title"><WarningIcon /> {{ $t( 'donation_confirmation_topbox_payment_title_bank_transfer_alt' ) }}</h1>
 			</div>
-			<div class="column is-half pt-0 pb-0 mt-0 mb-0">
+			<div class="column bank-data-content">
+				<h2> {{ $t( 'donation_confirmation_payment_bank_transfer_details' ) }} </h2>
+				<BankData class="no-margin" :bank-transfer-code="donation.bankTransferCode" :bank-transfer-amount="donation.amount" />
+			</div>
+			<div class="column pt-0 pb-0 mt-0 mb-0">
```

Comment:
The column on the right also needs the `is-half` some padding top:

```suggestion
			<div class="column is-half pt-18 pb-0 mt-0 mb-0">
```